

===   [ 收集测试用例 ]  === 


== cases\__st__.py 


== cases\blank_env\__st__.py 


== cases\blank_env\loginAPI.py 


== cases\blank_env\customerAPI\addCustomer.py 


== cases\blank_env\customerAPI\delCustomer.py 


== cases\blank_env\customerAPI\listCustomer.py 


== cases\blank_env\customerAPI\modifyCustomer.py 


== cases\blank_env\medicineAPI\addMedicine.py 


== cases\blank_env\medicineAPI\delMedicine.py 


== cases\blank_env\medicineAPI\listMedicine.py 


== cases\blank_env\medicineAPI\modifyMedicine.py 


== cases\blank_env\oneCustomer_env\__st__.py 


== cases\blank_env\oneCustomer_env\delCustomer.py 


== cases\blank_env\oneCustomer_env\modifyCustomer.py 


== cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\__st__.py 


== cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\delCustomer.py 


== cases\blank_env\oneCustomer_env\twoCustomers_env\__st__.py 


== cases\blank_env\oneCustomer_env\twoCustomers_env\modifyCustomer.py 


== cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\__st__.py 


== cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\addCustomer.py 


== cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py 


== cases\blank_env\oneMedicine_env\__st__.py 


== cases\blank_env\oneMedicine_env\delMedicine.py 


== cases\blank_env\oneMedicine_env\modifyMedicine.py 


== cases\blank_env\oneMedicine_env\twoMedicine_env\__st__.py 


== cases\blank_env\oneMedicine_env\twoMedicine_env\modifyMedicine.py 



===   [ 执行测试用例 ]  === 

预备执行用例数量 : 63



  ========= 测试开始 : 20250505_214729 =========



>>> cases\

[ suite setup ] cases\


>>> cases\blank_env\

[ suite setup ] cases\blank_env\

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Set-Cookie: sessionid=0i2gox5h46vzyg7fid3wcz8i1z347s3g; Path=/; Max-Age=1209600; HttpOnly
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除所有订单
列出订单

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 32

{"ret":0,"retlist":[],"total":0}
-------- HTTP response * end -------

删除所有客户
列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 32

{"ret":0,"retlist":[],"total":0}
-------- HTTP response * end -------

所有药品
列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 32

{"ret":0,"retlist":[],"total":0}
-------- HTTP response * end -------



>>> cases\blank_env\loginAPI.py

* 登录-0001  -  2025-05-05 21:47:29

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Set-Cookie: sessionid=2vii7sbbt4henoq38447w4s0lm4rhag6; Path=/; Max-Age=1209600; HttpOnly
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0002  -  2025-05-05 21:47:29

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0003  -  2025-05-05 21:47:29

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:29 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0004  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0005  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0006  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0007  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  通过

  PASS 

* 登录-0008  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\loginAPI.py", line 68, in teststeps
    CHECK_POINT('检查返回值', res == ret.json())
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 登录-0009  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 登录系统 

登录系统

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 45

{"msg":"用户名或者密码错误","ret":1}
-------- HTTP response * end -------


** 检查点 **  检查返回值 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\loginAPI.py", line 68, in teststeps
    CHECK_POINT('检查返回值', res == ret.json())
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\customerAPI\addCustomer.py

* 添加客户-API-0151  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 18

{"id":532,"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 141

{"ret":0,"retlist":[{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  通过

  PASS 

* 添加客户-API-0153  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 125

{"msg":"req body format error\nKey: 'Body.Data.Name' Error:Field validation for 'Name' failed on the 'required' tag","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 141

{"ret":0,"retlist":[{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\addCustomer.py", line 59, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加客户-API-0154  -  2025-05-05 21:47:30

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:30 GMT
Content-Length: 18

{"id":533,"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\addCustomer.py", line 81, in teststeps
    CHECK_POINT('返回的ret值', addRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加客户-API-0155  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 139

{"msg":"req body format error\nKey: 'Body.Data.Phonenumber' Error:Field validation for 'Phonenumber' failed on the 'required' tag","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\addCustomer.py", line 124, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加客户-API-0156  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 18

{"id":534,"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\addCustomer.py", line 146, in teststeps
    CHECK_POINT('返回的ret值', addRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\customerAPI\delCustomer.py

* 删除客户-API-0251  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 删除一个不存在ID的客户 

删除客户：id=100

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\delCustomer.py", line 13, in teststeps
    CHECK_POINT('返回的ret值=2', addRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\customerAPI\listCustomer.py

* 列出客户-API-0101  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 尝试列出客户，没有携带有效的sessionid 

列出客户

-------- HTTP response * begin -------
307
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 57

{"msg":"未登录","redirect":"/mgr/sign.html","ret":302}
-------- HTTP response * end -------

返回状态码是307
listRet{'msg': '未登录', 'redirect': '/mgr/sign.html', 'ret': 302}

** 检查点 **  返回的消息体数据正确 ---->  通过

  PASS 

* 列出客户-API-0102  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  检查状态码 ---->  通过


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 42, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0103  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 35

{"msg":"bad pagenum:hello","ret":2}
-------- HTTP response * end -------

状态码:400
listRet{'msg': 'bad pagenum:hello', 'ret': 2}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 60, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0104  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 36

{"msg":"bad pagesize:hello","ret":2}
-------- HTTP response * end -------

状态码:400
listRet{'msg': 'bad pagesize:hello', 'ret': 2}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 78, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0105  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------

状态码:200
listRet:{'ret': 0, 'retlist': [{'address': '南京市鼓楼北路', 'id': 534, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}, {'address': '南京市鼓楼北路', 'id': 533, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}, {'address': '武汉市桥西医院北路', 'id': 532, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}], 'total': 3}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 96, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0106  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------

状态码:200
listRet:{'ret': 0, 'retlist': [{'address': '南京市鼓楼北路', 'id': 534, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}], 'total': 3}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 113, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0107  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------

状态码:200
listRet:{'ret': 0, 'retlist': [{'address': '南京市鼓楼北路', 'id': 534, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}, {'address': '南京市鼓楼北路', 'id': 533, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}, {'address': '武汉市桥西医院北路', 'id': 532, 'name': '武汉市桥西医院', 'phonenumber': '13345679934'}], 'total': 3}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\listCustomer.py", line 130, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\customerAPI\modifyCustomer.py

* 修改客户-API-0201  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 修改一个不存在ID的客户 

修改客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 39

{"msg":"customer id not exist","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\customerAPI\modifyCustomer.py", line 25, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\medicineAPI\addMedicine.py

* 添加药品-API-0351  -  2025-05-05 21:47:31

  [ case execution steps ]

-- 第 1 步 -- 添加一个药品 

添加药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 18

{"id":123,"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:31 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  通过

  PASS 

* 添加药品-API-0353  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 添加一个药品 

添加药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 125

{"msg":"req body format error\nKey: 'Body.Data.Name' Error:Field validation for 'Name' failed on the 'required' tag","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\addMedicine.py", line 56, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加药品-API-0354  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 添加一个药品 

添加药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 125

{"msg":"req body format error\njson: cannot unmarshal string into Go struct field Body.data of type apiMgr.Medicine","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\addMedicine.py", line 87, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加药品-API-0355  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 添加一个药品 

添加药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 121

{"msg":"req body format error\nKey: 'Body.Data.Sn' Error:Field validation for 'Sn' failed on the 'required' tag","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\addMedicine.py", line 112, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 添加药品-API-0356  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 添加一个药品 

添加药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 125

{"msg":"req body format error\njson: cannot unmarshal string into Go struct field Body.data of type apiMgr.Medicine","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\addMedicine.py", line 143, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\medicineAPI\delMedicine.py

* 删除药品-API-0451  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 删除一个不存在ID的客户 

删除药品：id=100

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\delMedicine.py", line 13, in teststeps
    CHECK_POINT('返回的ret值=2', addRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\medicineAPI\listMedicine.py

* 列出药品-API-0301  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 尝试列出药品，没有携带有效的sessionid 

列出药品

-------- HTTP response * begin -------
307
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 57

{"msg":"未登录","redirect":"/mgr/sign.html","ret":302}
-------- HTTP response * end -------

返回状态码是307
listRet{'msg': '未登录', 'redirect': '/mgr/sign.html', 'ret': 302}

** 检查点 **  返回的消息体数据正确 ---->  通过

  PASS 

* 列出药品-API-0302  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 尝试列出药品，没有携带有效的sessionid 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------

返回状态码是200

** 检查点 **  检查状态码 ---->  通过

listRet{'ret': 0, 'retlist': [{'desc': '青霉素 国字号', 'id': 123, 'name': '青霉素', 'sn': '099877883837'}], 'total': 1}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 41, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出药品-API-0303  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 列出药品 

列出药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 35

{"msg":"bad pagenum:hello","ret":2}
-------- HTTP response * end -------

返回状态码是400
listRet{'msg': 'bad pagenum:hello', 'ret': 2}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 58, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出药品-API-0304  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 列出药品 

列出药品

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 36

{"msg":"bad pagesize:hello","ret":2}
-------- HTTP response * end -------

返回状态码是400
listRet:{'msg': 'bad pagesize:hello', 'ret': 2}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 76, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出药品-API-0305  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 列出药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------

返回状态码是200
listRet:{'ret': 0, 'retlist': [{'desc': '青霉素 国字号', 'id': 123, 'name': '青霉素', 'sn': '099877883837'}], 'total': 1}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 94, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出药品-API-0306  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 列出药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------

返回状态码是200
listRet:{'ret': 0, 'retlist': [{'desc': '青霉素 国字号', 'id': 123, 'name': '青霉素', 'sn': '099877883837'}], 'total': 1}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 112, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出药品-API-0307  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 列出药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------

返回状态码是200
listRet:{'ret': 0, 'retlist': [{'desc': '青霉素 国字号', 'id': 123, 'name': '青霉素', 'sn': '099877883837'}], 'total': 1}

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\listMedicine.py", line 130, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\medicineAPI\modifyMedicine.py

* 修改药品-API-0401  -  2025-05-05 21:47:32

  [ case execution steps ]

-- 第 1 步 -- 修改一个不存在ID的药品 

修改客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 39

{"msg":"medicine id not exist","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:32 GMT
Content-Length: 110

{"ret":0,"retlist":[{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":1}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\medicineAPI\modifyMedicine.py", line 24, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneCustomer_env\

[ suite setup ] cases\blank_env\oneCustomer_env\
添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 18

{"id":535,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneCustomer_env\delCustomer.py

* 删除客户-API-0252  -  2025-05-05 21:47:33

  [ case execution steps ]
列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 143

{"ret":0,"retlist":[{"address":"武汉市桥西医院北路1","id":535,"name":"武汉市桥西医院1","phonenumber":"13345679901"}],"total":4}
-------- HTTP response * end -------


-- 第 1 步 -- 删除客户：id为535 

删除客户：id=535

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\delCustomer.py", line 35, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneCustomer_env\modifyCustomer.py

* 修改客户-API-0202  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"南京市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 43, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0203  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"南京市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"南京市桥西医院","phonenumber":"13923567443"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 74, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0204  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 135

{"ret":0,"retlist":[{"address":"南京市鼓楼北路","id":534,"name":"南京市桥西医院","phonenumber":"13923567443"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 138

{"ret":0,"retlist":[{"address":"镇江市桃花坞123号","id":534,"name":"南京市桥西医院","phonenumber":"13923567443"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 105, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0205  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 138

{"ret":0,"retlist":[{"address":"镇江市桃花坞123号","id":534,"name":"南京市桥西医院","phonenumber":"13923567443"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞123号","id":534,"name":"镇江市中医院","phonenumber":"1362345887"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 136, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0206  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞123号","id":534,"name":"镇江市中医院","phonenumber":"1362345887"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1362345887"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 167, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0207  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1362345887"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 198, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改客户-API-0208  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 修改客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":3}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\modifyCustomer.py", line 229, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\

[ suite setup ] cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\
添加药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 18

{"id":124,"ret":0}
-------- HTTP response * end -------

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":3}
-------- HTTP response * end -------

添加订单

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 17

{"id":37,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\delCustomer.py

* 删除客户-API-0253  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 删除一个已绑定订单的客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 134

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":3}
-------- HTTP response * end -------

删除客户：id=534

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 72

{"msg":"删除记录失败，或许需要先删除相关订单","ret":2}
-------- HTTP response * end -------


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 348

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\delCustomer.py", line 40, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


[ suite teardown ] cases\blank_env\oneCustomer_env\oneCustomerOneMedicineOneOrder\


>>> cases\blank_env\oneCustomer_env\twoCustomers_env\

[ suite setup ] cases\blank_env\oneCustomer_env\twoCustomers_env\
添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 18

{"id":536,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneCustomer_env\twoCustomers_env\modifyCustomer.py

* 修改客户-API-0209  -  2025-05-05 21:47:33

  [ case execution steps ]

-- 第 1 步 -- 获取修改客户的ID 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 143

{"ret":0,"retlist":[{"address":"武汉市桥西医院北路2","id":536,"name":"武汉市桥西医院2","phonenumber":"13345679902"}],"total":4}
-------- HTTP response * end -------


-- 第 2 步 -- 修改客户:修改name与已经存在的一样 

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=1 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\modifyCustomer.py", line 30, in teststeps
    CHECK_POINT('返回的ret值=1', addRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\

[ suite setup ] cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\
添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 18

{"id":537,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:33 GMT
Content-Length: 18

{"id":538,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":539,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":540,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":541,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":542,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":543,"ret":0}
-------- HTTP response * end -------

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":544,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\addCustomer.py

* 添加客户-API-0152  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 添加一个客户 

添加客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":545,"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 1255

{"ret":0,"retlist":[{"address":"南京市桥东医院北路","id":545,"name":"南京市桥东医院","phonenumber":"12245789934"},{"address":"武汉市桥西医院北路10","id":544,"name":"武汉市桥西医院10","phonenumber":"13345679910"},{"address":"武汉市桥西医院北路9","id":543,"name":"武汉市桥西医院9","phonenumber":"13345679909"},{"address":"武汉市桥西医院北路8","id":542,"name":"武汉市桥西医院8","phonenumber":"13345679908"},{"address":"武汉市桥西医院北路7","id":541,"name":"武汉市桥西医院7","phonenumber":"13345679907"},{"address":"武汉市桥西医院北路6","id":540,"name":"武汉市桥西医院6","phonenumber":"13345679906"},{"address":"武汉市桥西医院北路5","id":539,"name":"武汉市桥西医院5","phonenumber":"13345679905"},{"address":"武汉市桥西医院北路4","id":538,"name":"武汉市桥西医院4","phonenumber":"13345679904"},{"address":"武汉市桥西医院北路3","id":537,"name":"武汉市桥西医院3","phonenumber":"13345679903"},{"address":"武汉市桥西医院北路2","id":536,"name":"武汉市桥西医院1","phonenumber":"13345679902"},{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"}],"total":13}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\addCustomer.py", line 45, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py

* 列出客户-API-0108  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 1152

{"ret":0,"retlist":[{"address":"南京市桥东医院北路","id":545,"name":"南京市桥东医院","phonenumber":"12245789934"},{"address":"武汉市桥西医院北路10","id":544,"name":"武汉市桥西医院10","phonenumber":"13345679910"},{"address":"武汉市桥西医院北路9","id":543,"name":"武汉市桥西医院9","phonenumber":"13345679909"},{"address":"武汉市桥西医院北路8","id":542,"name":"武汉市桥西医院8","phonenumber":"13345679908"},{"address":"武汉市桥西医院北路7","id":541,"name":"武汉市桥西医院7","phonenumber":"13345679907"},{"address":"武汉市桥西医院北路6","id":540,"name":"武汉市桥西医院6","phonenumber":"13345679906"},{"address":"武汉市桥西医院北路5","id":539,"name":"武汉市桥西医院5","phonenumber":"13345679905"},{"address":"武汉市桥西医院北路4","id":538,"name":"武汉市桥西医院4","phonenumber":"13345679904"},{"address":"武汉市桥西医院北路3","id":537,"name":"武汉市桥西医院3","phonenumber":"13345679903"},{"address":"武汉市桥西医院北路2","id":536,"name":"武汉市桥西医院1","phonenumber":"13345679902"}],"total":13}
-------- HTTP response * end -------

返回状态码是200
列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 142

{"ret":0,"retlist":[{"address":"南京市桥东医院北路","id":545,"name":"南京市桥东医院","phonenumber":"12245789934"}],"total":13}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py", line 45, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0109  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":13}
-------- HTTP response * end -------

返回状态码是200

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py", line 66, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0110  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 33

{"ret":0,"retlist":[],"total":13}
-------- HTTP response * end -------

返回状态码是200

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py", line 87, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0111  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 349

{"ret":0,"retlist":[{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":13}
-------- HTTP response * end -------

返回状态码是200

** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py", line 108, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 列出客户-API-0112  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 32

{"ret":0,"retlist":[],"total":0}
-------- HTTP response * end -------

返回状态码是200

** 检查点 **  返回的消息体数据正确 ---->  通过

  PASS 

* 列出客户-API-0113  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 列出客户 

列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 1146

{"ret":0,"retlist":[{"address":"武汉市桥西医院北路10","id":544,"name":"武汉市桥西医院10","phonenumber":"13345679910"},{"address":"武汉市桥西医院北路9","id":543,"name":"武汉市桥西医院9","phonenumber":"13345679909"},{"address":"武汉市桥西医院北路8","id":542,"name":"武汉市桥西医院8","phonenumber":"13345679908"},{"address":"武汉市桥西医院北路7","id":541,"name":"武汉市桥西医院7","phonenumber":"13345679907"},{"address":"武汉市桥西医院北路6","id":540,"name":"武汉市桥西医院6","phonenumber":"13345679906"},{"address":"武汉市桥西医院北路5","id":539,"name":"武汉市桥西医院5","phonenumber":"13345679905"},{"address":"武汉市桥西医院北路4","id":538,"name":"武汉市桥西医院4","phonenumber":"13345679904"},{"address":"武汉市桥西医院北路3","id":537,"name":"武汉市桥西医院3","phonenumber":"13345679903"},{"address":"武汉市桥西医院北路2","id":536,"name":"武汉市桥西医院1","phonenumber":"13345679902"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":11}
-------- HTTP response * end -------

返回状态码是200
列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 142

{"ret":0,"retlist":[{"address":"南京市桥东医院北路","id":545,"name":"南京市桥东医院","phonenumber":"12245789934"}],"total":13}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\listCustomer.py", line 162, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


[ suite teardown ] cases\blank_env\oneCustomer_env\twoCustomers_env\tenCustomers_env\

[ suite teardown ] cases\blank_env\oneCustomer_env\twoCustomers_env\

[ suite teardown ] cases\blank_env\oneCustomer_env\
删除所有订单
列出订单

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 222

{"ret":0,"retlist":[{"create_date":"2025-05-05T13:47:33Z","customer_name":"镇江市中医院","customerid":534,"id":37,"medicinelist":"[{\"id\":124,\"name\":\"青霉素1\",\"amount\":\"5\"}]","name":"订单1"}],"total":1}
-------- HTTP response * end -------

删除订单：id=37

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除所有客户
列出客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 1469

{"ret":0,"retlist":[{"address":"南京市桥东医院北路","id":545,"name":"南京市桥东医院","phonenumber":"12245789934"},{"address":"武汉市桥西医院北路10","id":544,"name":"武汉市桥西医院10","phonenumber":"13345679910"},{"address":"武汉市桥西医院北路9","id":543,"name":"武汉市桥西医院9","phonenumber":"13345679909"},{"address":"武汉市桥西医院北路8","id":542,"name":"武汉市桥西医院8","phonenumber":"13345679908"},{"address":"武汉市桥西医院北路7","id":541,"name":"武汉市桥西医院7","phonenumber":"13345679907"},{"address":"武汉市桥西医院北路6","id":540,"name":"武汉市桥西医院6","phonenumber":"13345679906"},{"address":"武汉市桥西医院北路5","id":539,"name":"武汉市桥西医院5","phonenumber":"13345679905"},{"address":"武汉市桥西医院北路4","id":538,"name":"武汉市桥西医院4","phonenumber":"13345679904"},{"address":"武汉市桥西医院北路3","id":537,"name":"武汉市桥西医院3","phonenumber":"13345679903"},{"address":"武汉市桥西医院北路2","id":536,"name":"武汉市桥西医院1","phonenumber":"13345679902"},{"address":"镇江市桃花坞666号","id":534,"name":"镇江市中医院","phonenumber":"1275678433"},{"address":"南京市鼓楼北路","id":533,"name":"武汉市桥西医院","phonenumber":"13345679934"},{"address":"武汉市桥西医院北路","id":532,"name":"武汉市桥西医院","phonenumber":"13345679934"}],"total":13}
-------- HTTP response * end -------

删除客户：id=545

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=544

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=543

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=542

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=541

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=540

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=539

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=538

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=537

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=536

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=534

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=533

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除客户：id=532

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneMedicine_env\

[ suite setup ] cases\blank_env\oneMedicine_env\
添加药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 18

{"id":125,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneMedicine_env\delMedicine.py

* 删除药品-API-0452  -  2025-05-05 21:47:34

  [ case execution steps ]
列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 112

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":125,"name":"青霉素1","sn":"099877883801"}],"total":3}
-------- HTTP response * end -------


-- 第 1 步 -- 删除药品：id为125 

删除药品：id=125

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 112

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"青霉素1","sn":"099877883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\delMedicine.py", line 35, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneMedicine_env\modifyMedicine.py

* 修改药品-API-0402  -  2025-05-05 21:47:34

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 112

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"青霉素1","sn":"099877883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:34 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"叶黄素","sn":"099877883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 43, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0403  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"叶黄素","sn":"099877883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 74, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0404  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"青霉素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 114

{"ret":0,"retlist":[{"desc":"青霉素 江苏字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 105, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0405  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 114

{"ret":0,"retlist":[{"desc":"青霉素 江苏字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 114

{"ret":0,"retlist":[{"desc":"青霉素 江苏字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 136, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0406  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 114

{"ret":0,"retlist":[{"desc":"青霉素 江苏字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 167, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0407  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 198, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


* 修改药品-API-0408  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 修改药品 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------

修改客户

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=0 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 111

{"ret":0,"retlist":[{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":2}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\modifyMedicine.py", line 229, in teststeps
    CHECK_POINT('返回的消息体数据正确', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError



>>> cases\blank_env\oneMedicine_env\twoMedicine_env\

[ suite setup ] cases\blank_env\oneMedicine_env\twoMedicine_env\
添加药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 18

{"id":126,"ret":0}
-------- HTTP response * end -------



>>> cases\blank_env\oneMedicine_env\twoMedicine_env\modifyMedicine.py

* 修改药品-API-0409  -  2025-05-05 21:47:35

  [ case execution steps ]

-- 第 1 步 -- 获取修改药品的ID 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 112

{"ret":0,"retlist":[{"desc":"青霉素 国字号2","id":126,"name":"青霉素2","sn":"099877883802"}],"total":3}
-------- HTTP response * end -------


-- 第 2 步 -- 修改药品:修改药品与已经存在的一样 

修改客户

-------- HTTP response * begin -------
400
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 39

{"msg":"customer id not exist","ret":2}
-------- HTTP response * end -------


** 检查点 **  返回的ret值=2 ---->  通过


-- 第 2 步 -- 检查系统数据 

列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 192

{"ret":0,"retlist":[{"desc":"青霉素 国字号2","id":126,"name":"青霉素2","sn":"099877883802"},{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"}],"total":3}
-------- HTTP response * end -------


** 检查点 **  返回的消息体数据正确，未成功修改 ---->  !! 不通过!!

  FAIL    
Traceback:
  File "D:\tools\projects\autotest_bysms\cases\blank_env\oneMedicine_env\twoMedicine_env\modifyMedicine.py", line 48, in teststeps
    CHECK_POINT('返回的消息体数据正确，未成功修改', listRet == expected)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\site-packages\hytest\common.py", line 76, in CHECK_POINT
    raise AssertionError()
AssertionError


[ suite teardown ] cases\blank_env\oneMedicine_env\twoMedicine_env\

[ suite teardown ] cases\blank_env\oneMedicine_env\
删除所有订单
列出订单

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 32

{"ret":0,"retlist":[],"total":0}
-------- HTTP response * end -------

所有药品
列出药品

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 271

{"ret":0,"retlist":[{"desc":"青霉素 国字号2","id":126,"name":"青霉素2","sn":"099877883802"},{"desc":"叶黄素 国字号1","id":124,"name":"叶黄素","sn":"091234883801"},{"desc":"青霉素 国字号","id":123,"name":"青霉素","sn":"099877883837"}],"total":3}
-------- HTTP response * end -------

删除药品：id=126

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除药品：id=124

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------

删除药品：id=123

-------- HTTP response * begin -------
200
Content-Type: application/json; charset=utf-8
Date: Mon, 05 May 2025 13:47:35 GMT
Content-Length: 9

{"ret":0}
-------- HTTP response * end -------


[ suite teardown ] cases\blank_env\

[ suite teardown ] cases\


  ========= 测试结束 : 20250505_214735 =========


  耗时    : 6.006 秒


  预备执行用例数量 : 63

  实际执行用例数量 : 63

  通过 : 12

  失败 : 51

  异常 : 0

  套件初始化失败 : 0

  套件清除  失败 : 0

  用例初始化失败 : 0

  用例清除  失败 : 0
